boards/raspberry_pi_pico/Makefile

```diff
@@ -32,5 +30,3 @@ ifeq ($(APP),)
 	$(error Please define the APP variable with the TBF file to flash an application) 
 endif 
 	arm-none-eabi-objcopy --update-section .apps=$(APP) $(KERNEL) $(KERNEL_WITH_APP)
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $(KERNEL_WITH_APP); verify_image $(KERNEL_WITH_APP); reset; shutdown;"
```

Comment:
?

Reply:
The Pico can be flashed using a custom version of openocd and another Raspberry Pi.

Reply:
But if `make install` doesn't install then we should delete the target.